contentcuration/contentcuration/frontend/accounts/router.js

```diff
@@ -0,0 +1,28 @@
+import VueRouter from 'vue-router';
+import Main from './pages/Main.vue';
+import Create from './pages/Create.vue';
+import ActivationSent from './pages/ActivationSent.vue';
+import ForgotPassword from './pages/ForgotPassword.vue';
+
+function pageRoute(path, component) {
```

Comment:
We can keep this for the login page at least, but for this first pass on the vue migration, we might run into difficulties supporting this for things like the edit modal where there are [multiple entry points](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/frontend/channelEdit/router.js#L69). We are planning to move away from this eventually, but I'd rather we discuss adding this convention then as well to make sure we don't jump the gun here

Reply:
yeah this is just a convenience function. It's a thin wrapper that can be bypassed easily